blueprints/showcase-plugin-with-media/blueprint.json

```
@@ -0,0 +1,102 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Showcase plugin",
+		"description": "Offer full demo of plugin with media included",
+		"author": "lovor",
```

The author maps to the GitHub username. 

```suggestion
		"author": "Lovor01",
```

---

I thought the paths are supposed to be final paths in blueprint repository, not the paths for temporary fork. Ok, I will change them.